src/window.rs

```diff
@@ -304,6 +304,10 @@ impl Window {
     ///
     /// See the [`dpi`](dpi/index.html) module for more information.
     ///
+    /// Note that this value can change depending on user action (for example if the window is
+    /// moved to another screen), as such tracking the `WindowEvent::HiDpiFactorChanged` events is
```

Comment:
So, I was wondering if "as such" here ought to be preceded by a semicolon rather than a comma, but after doing some digging, it turns out that "as such" is quite the can of worms... https://www.chronicle.com/blogs/linguafranca/2015/04/24/academic-writing-as-such/

Since I use "as such" rather often, it will take a lot of soul searching before I can decide how I feel about it... ~~as such~~, I'm not going to make a fuss about this. I mostly just found it interesting.

(Though preceding with a semicolon + putting a comma after "as such" is probably still a good idea?)